Tests/SwiftGRPCTests/ClientTestExample.swift

```diff
@@ -92,6 +98,30 @@ extension ClientTestExample {
     XCTAssertEqual([Echo_EchoRequest(text: "foo")], fakeService.getRequests)
   }
   
+  func testClientAsynchronous() throws {
+    
+    let fakeService = Echo_EchoServiceTestStub()
+    fakeService.getResponses.append(Echo_EchoResponse(text: "bar"))
+    
+    let client = ClientUnderTest(service: fakeService)
+    
+    let completionHandlerExpectation = expectation(description: "final completion handler called")
+
+    var wordAsynchronous:String!
+    try client.getWordasynchronous("foo",completion:{ (word:String) in
+        wordAsynchronous = word
+        completionHandlerExpectation.fulfill()
+    })
+    
+    self.wait(for: [completionHandlerExpectation], timeout: 1)
```

Comment:
Use `waitForExpectations(timeout: defaultTimeout)` instead, to fix compilation under Linux.